pkg/importer/dataStream.go

```diff
@@ -183,6 +183,7 @@ func (d dataStream) http() (io.ReadCloser, error) {
 
 func (d dataStream) local() (io.ReadCloser, error) {
 	fn := d.Url.Path
+	glog.Warningf("attempting to import local file %q which is not officially supported", fn)
```

Comment:
Can this be a little more verbose?  Something like
> Warning - the file:/// protocol is intended for debug and testing only as it cannot access filesystems external to the container.

Or something along those lines?